tests/test_practitioner.py

```diff
@@ -270,3 +270,9 @@ def test_practitioner_put(self):
                                data=json.dumps(data),
                                content_type='application/json')
         self.assertEqual(resp.status_code, 409)
+
+        # test updating with same external identifier
+        resp = self.client.put('/api/practitioner/{}'.format(pract2_id),
```

Comment:
Wouldn't it be best to test using the external identifier in the URL?

Reply:
Sure. Those are really two different sections of the PUT code, and the external identifier query part is tested above, but I can also test it here as well